validation-test/compiler_crashers_fixed/28385-swift-constraints-constraintgraph-addconstraint.swift

```diff
@@ -5,6 +5,5 @@
 // See http://swift.org/LICENSE.txt for license information
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 
-// RUN: not --crash %target-swift-frontend %s -parse
-// REQUIRES: asserts
+// RUN: not %target-swift-frontend %s -parse
 assert((n:String{
```

Comment:
It produces 'expression type '(n: _)' is ambiguous without more context' now, which sucks, but better than a crash at least.